### PR TITLE
EP-1: Create wrapper for Sia package

### PR DESCRIPTION
**Pull request by a-ruban**
Done:

- [x] Create wrapper for Sia package;

- [x] Refactor package code due to code-style.

### CODE REVIEW

File: sia/endpoints/host.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
+"""
+Host API.
+"""
+
```
Comment (dmytrostriletskyi) [inline sia/endpoints/host.py]: No blank line.

File: sia/endpoints/miner.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+"""
+from sia.constants import miner as miner_constants
+from sia.mixins.http import Http
+
+
+class Miner:
+    """
+    Miner API.
+    """
+    def __init__(self):
```
Comment (dmytrostriletskyi) [inline sia/endpoints/miner.py]: Blank line.

**Review by dmytrostriletskyi — CHANGES_REQUESTED** [review]

File: sia/endpoints/miner.py (hunk 2 above)
Comment (a-ruban) [reply sia/endpoints/miner.py]: @dmytrostriletskyi done.

File: sia/endpoints/host.py (hunk 1 above)
Comment (a-ruban) [reply sia/endpoints/host.py]: @dmytrostriletskyi done.

**Review by kozlov-oleksandr — APPROVED** [review]

**Review by dmytrostriletskyi — APPROVED** [review]

**Pull request merged**